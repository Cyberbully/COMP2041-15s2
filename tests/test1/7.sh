#!/bin/sh

# Enter pipeline/shell script below.
for filename in *.2041
do
    head -1 "$filename"
done
