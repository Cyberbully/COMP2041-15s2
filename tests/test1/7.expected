yxLAqn HtIeykbLKHLmhwNnvKmIviYHTFLQPyxhyrfiRFANLSgZOmvCGCoRAslvFylNDIZDcXCpSSMnhEMNOAGUGNSbUqVWY
PVkokgCvfUS sUkdTw
BvcTjnfgmGjtRbckFQYadHDUQqalKAKJPIYrj
PZRRtrxThdIcHzGkAxRWUKLyYavAObKKdwxenTvKNmtlfJIPUhpZCmDKJlZy cIbLNzFgPPxKBI
ffwmew UhjsISXhrNvbFLjcneb QBXSJyVGaaNeRZSBeoZRdne
aHDBygqqSQXl fSAYMEmePTotibNPiwMryiOr bq
ZMpALVxVmZjVyfianvQdPJkGhB HbtbWJc
qMMOYVkbrEMasf
tSyuTZctSDAfGfqiYnKJFGvfnuluexUmkyBbyKWQbNLxmRCJRkXxkac  PcDamLUCeFBUvyPLu sZqnQULIXrzBiNJA
GyJblAq RDLZPFmVerhXPasGVRbZFSvJJMPhBXiPhttVpRWuJfybwCLdCDrxPHTFQkvIFKRmwNhFNBulEf TgZKs kuQM
IgaQiDWWuqRlQFgYeayAGCYIlqYw RcsGJHoydShTafqzpmLcclfdjVYHBoDBFXHVVxzXBgRCrJbooGseBAguMPmKSbLtLc
